### PR TITLE
Add Blank starter project to docs

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
Added the entry to the TOC and wrote some stuff. Please read to make sure it sounds fine 

Welcome page:
image

Blank project overview:
image

ToC:
image

#### Tests
`improbadoc serve ./docs`

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

**Comment by oblm** [comment]
I'd love to see:
1. A mention of the blank starter project in: `/docs/welcome.md` alongside the FPS Starter Project.
2. A mention of the blank starter project in: README.md
3. For the blank starter project to appears in the Starter Projects section of the docs TOC underneath the FPS Starter Project.

File: docs/projects/blank/overview.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
 
-// Information about the blank template project
+The Blank starter project is a great starting point for working on the SpatialOS GDK for Unity if you want a clean slate.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: Is it called "Blank Starter Project" (proper noun) or "Blank starter project"?

The "FPS Starter Project" is a proper noun in all the docs (except, I see where it isn't - I didn't get to look at the docs in the project directory!)

It's important to be consistent as this aids user understanding - they aren't puzzling over whether we mean the same thing when we write the name of this.

In addition, if it's a proper noun, keep capital initials in the title. If it's not, remove them as our title standard is newspaper titles style (like a sentence but without a full-stop).

File: docs/projects/blank/overview.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: There's quite a few bits on here so have them all in one to make it easier to pop in:

```
# Blank Starter Project

The Blank Starter Project is a great starting point to working on the SpatialOS GDK for Unity if you want a clean slate.

Check out the [Blank Starter Project on Github](https://github.com/spatialos/gdk-for-unity-blank-project).

## What does it come with?

It has the minimum GDK feature set you need to start developing games for SpatialOS:

- A Unity project set up with the SpatialOS GDK for Unity.
- Sample [local and cloud deployment]({{urlRoot}}/contents/glossary#deploying) configurations.
- An empty snapshot.
- [Client-worker and server-worker]({{urlRoot}}/contents/glossary#worker) JSON [configuration files (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#worker-attribute-sets).
- Client-worker and server-worker [worker connectors]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector and [worker prefabs]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector#how-to-use-worker-prefabs).
- [Worker build configuration]({{urlRoot}}/content/build#build-your-workers).
- A set of Unity Scenes to use in development.

## Licensing

The Blank starter project is shared under the Improbable license; you can read more about what that entails in our [licensing]({{urlRoot}}/license) section.
```

File: docs/projects/blank/overview.md (hunk 1 above)
Comment (ElleEss) [reply docs/projects/blank/overview.md]: Two "fors" in the sentence make it hard to read. "Clean slate" is an English-language colloquialism which may not mean anything to non-US and non-UK users.

File: docs/projects/blank/overview.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
 
-// Information about the blank template project
+The Blank starter project is a great starting point for working on the SpatialOS GDK for Unity if you want a clean slate.
+
+Check out the [Blank starter project on Github](https://github.com/spatialos/gdk-for-unity-blank-project)!
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: Lose the exclamation mark, we have no character in docs. Life is serious.

Also, suggest "Find the..."

File: docs/projects/blank/overview.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+## What does it come with?
+
+It has the minimal feature set required to start developing:
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: "minimum" better than "minimal" - "minimum" means the absolute smallest amount so works with the rest of the sentence. Minimal means a small amount and doesn't work with the rest of the sentence.

File: docs/projects/blank/overview.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+
+- A Unity project setup with the SpatialOS GDK for Unity.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: set up  (it's a verb here, rather than a noun).

File: docs/projects/blank/overview.md [hunk 6]
```diff
@@ -1,3 +1,21 @@
...
+- Sample local and cloud deployment configurations.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: Link to glossary

File: docs/projects/blank/overview.md [hunk 7]
```diff
@@ -1,3 +1,21 @@
...
+- An empty snapshot.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: Link to glossary.

File: docs/projects/blank/overview.md [hunk 8]
```diff
@@ -1,3 +1,21 @@
...
+- Client and server workers JSON configurations.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: "client-worker" and "server-worker" to use consistent terminology. (Changing it is very confusing for users who have a fragile grasp of our technology.)
Link to worker config docs.

File: docs/projects/blank/overview.md [hunk 9]
```diff
@@ -1,3 +1,21 @@
...
+- Client and server worker connectors and prefabs.
```
Comment (ElleEss) [inline docs/projects/blank/overview.md]: Link to worker connector docs (see above-edited version of this doc).
"Client-worker" and "server-worker" to maintain consistent terminology.

File: docs/projects/blank/overview.md (hunk 1 above)
Comment (ElleEss) [reply docs/projects/blank/overview.md]: (See above for edited version of this doc.)

File: docs/toc.md [hunk 10]
```diff
@@ -13,6 +13,7 @@
 ***
 - <h3> Starter projects</h3>
 - <h3> [ - FPS starter project]({{urlRoot}}/projects/fps/overview)</h3>
+- <h3> [ - Blank starter project]({{urlRoot}}/projects/blank/overview)</h3>
```
Comment (ElleEss) [inline docs/toc.md]: Just noticed that these should be proper nouns -adding on my version of the ToC.

Comment (ElleEss) [reply docs/toc.md]: I think ideally, we should mention this in the welcome page but I know that's a bit sensitive.

Comment (paulbalaji) [reply docs/toc.md]: updated 👍

File: docs/projects/blank/overview.md (hunk 3 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 4 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 5 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 6 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 7 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 8 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 9 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

File: docs/projects/blank/overview.md (hunk 2 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: included these changes

File: docs/projects/blank/overview.md (hunk 1 above)
Comment (paulbalaji) [reply docs/projects/blank/overview.md]: added full changes from above

**Comment by paulbalaji** [comment]
> 1. A mention of the blank starter project in: `/docs/welcome.md` alongside the FPS Starter Project.
> 2. A mention of the blank starter project in: README.md
> 3. For the blank starter project to appears in the Starter Projects section of the docs TOC underneath the FPS Starter Project.

1/2 - will need to discuss with team then come back here
3 - this is part of the PR

**Comment by paulbalaji** [comment]
updated PR description with preview images

**Pull request merged**